core/templates/dev/head/app.js

```diff
@@ -536,16 +535,24 @@ oppia.factory('rteHelperService', [
             explorationId: explorationContextService.getExplorationId()
           });
         }
-        var interpolatedUrl = $interpolate(
-          componentDefn.previewUrlTemplate, false, null, true)(
-          customizationArgsDict);
+        var componentPreviewUrlTemplate = componentDefn.previewUrlTemplate;
+        if (componentDefn.previewUrlTemplate.startsWith(
```

Comment:
Try `indexOf() === 0` here; startsWith is less browser-compatible. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith

Reply:
Done.